File: ws/bonilla/U1/WS10Constructors/ChickenFarmSimulatorJs/ec.espe.edu.view/ChickenFarmSimulator.js
import { Chicken } from "../ec.espe.edu.model/Chicken.js"

// Simulación del main
function main() {
  const id = 1;
  const name = "Lucy";
  const color = "white and brown";
  const age = 5;
  const molting = true;
  const bornOnDate = new Date(); // fecha actual

  const chicken = new Chicken(id, name, color, age, molting, bornOnDate);

  console.log("Edison's chicken data --> " + chicken.toString());
  console.log("Edison's chicken data -->");
  chicken.duStuff();

  console.log(`chicken --> ${chicken.getName()} is ${chicken.getColor()}`);

  chicken.setAge(12);
  console.log(`chicken --> ${chicken.getName()} is ${chicken.getAge()} years old`);
  console.log("Edison's chicken data --> " + chicken.toString());
}

// Ejecutar el programa
main();

File: ws/sivinta/U1/WS12ProjectStructure/ChickenFarmSimulatorControllerPY/ChickenFarmSimulator/ec/edu/espe/view/ChickenFarmSimulator.py
from ec.edu.espe.model.Chicken import Chicken
from datetime import date

if __name__ == "__main__":
    chicken_id = 1
    color = "white and brown"
    name = "Lucy"
    age = 5
    molting = True
    today = date.today()

    chicken = Chicken(chicken_id, name, color, age, molting, today)

    print(f"Fuentes's chicken data --> {chicken}")
    print("Fuentes's chicken data -->")
    chicken.do_stuff()

    print(f"chicken --> {chicken.name} is {chicken.color}")

    chicken.age = 12
    print(f"chicken --> {chicken.name} is {chicken.age} years-old")
    print(f"Fuentes's chicken data --> {chicken}")